apply/managementExampleOutput.json

```diff
@@ -47,33 +47,39 @@
             ],
             "textEquivalent": "Management Decision Aids",
             "resource": {
-              "reference": "CommunicationRequest/22"
+              "reference": "CommunicationRequest/28"
             }
           },
           {
-            "id": "18",
+            "id": "22",
             "title": "Recommended Management Activities",
             "description": "Recommended Management Activities",
             "textEquivalent": "Recommended Management Activities",
             "groupingBehavior": "visual-group",
             "selectionBehavior": "at-most-one",
+            "action": []
+          },
+          {
+            "id": "18",
+            "title": "Cervical Cancer Analytics Actions",
+            "description": "Top level action that defines trigger and criteria for inclusion and exclusion.",
+            "documentation": [
+              {
+                "type": "documentation",
+                "label": "Inclusions",
+                "display": "This PlanDefinition applies if RunAnalytics is set to true."
+              }
+            ],
+            "textEquivalent": "Static text equivalent of the action, used if the dynamic aspects cannot be interpreted by the receiving system",
```

Comment:
I don't know if I would update the Example Outputs quite yet, since Mike's PR (https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/166) is actually going to remove the Analytics PlanDef call from our screening and management PlanDefs

Reply:
This comment is still outstanding.

Reply:
I removed both Example Outputs in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/165/commits/116a4a41a00ae4ec2cff71f7341c79b5b429a664.